### PR TITLE
LibGfx+LibWeb: Fix css/css-color/hwb-004.html

### PR DESCRIPTION
**Pull request by LucasChollet**

### CODE REVIEW

**Comment by LucasChollet** [comment]
Oops, one change didn't make it to the final cut. And it seems that I'm too tired to figure out how to rebase the two other tests, I'll get back to it tomorrow.

File: Libraries/LibGfx/Color.h [hunk 1]
```diff
@@ -550,9 +550,9 @@ class Color {
             break;
         }
 
-        u8 out_r = (u8)(r * 255);
-        u8 out_g = (u8)(g * 255);
-        u8 out_b = (u8)(b * 255);
+        auto out_r = static_cast<u8>(round(r * 255));
+        auto out_g = static_cast<u8>(round(g * 255));
+        auto out_b = static_cast<u8>(round(b * 255));
```
Comment (gmta) [inline Libraries/LibGfx/Color.h:555]: Commit message: this is `::from_hsv()`, not `::with_opacity()`

File: Libraries/LibGfx/Color.h [hunk 2]
```diff
@@ -401,7 +401,7 @@ class Color {
 
     constexpr Color with_opacity(float opacity) const
     {
-        return with_alpha(alpha() * opacity);
+        return with_alpha(llround(clamp(alpha() * opacity, 0.0f, 255.0f)));
```
Comment (gmta) [inline Libraries/LibGfx/Color.h:404]: Why do you need `llround` if you clamp to `0..255`?

File: Libraries/LibWeb/CSS/StyleValues/CSSColorValue.cpp [hunk 3]
```diff
@@ -45,7 +45,7 @@ ValueComparingNonnullRefPtr<CSSColorValue> CSSColorValue::create_from_color(Colo
     return make_rgb_color(color);
```
Comment (gmta) [inline Libraries/LibWeb/CSS/StyleValues/CSSColorValue.cpp:45]: Commit message typo: laking

File: Libraries/LibGfx/Color.h (hunk 2 above)
Comment (gmta) [reply Libraries/LibGfx/Color.h:404]: Wouldn't `lround` do the same? I wouldn't mind `static_cast<u8>(...)` by the way, it makes it a bit more readable imo.

I guess values outside of 0..1 would be programming errors, couldn't hurt `VERIFY`ing!

Comment (LucasChollet) [reply Libraries/LibGfx/Color.h:404]: > Wouldn't lround do the same?

It would, now I just have to remember where I copied that from :sweat_smile:.

**Pull request merged**